packages/airnode-node/src/types.ts

```diff
@@ -287,11 +287,26 @@ export interface WorkerOptions {
   readonly stage: string;
 }
 
-export type WorkerFunctionName = 'initializeProvider' | 'callApi' | 'processProviderRequests';
+export interface InitializeProviderPayload {
+  readonly functionName: 'initializeProvider';
+  readonly state: ProviderState<EVMProviderState>;
+}
+
+export interface ProcessTransactionsPayload {
+  readonly functionName: 'processProviderRequests';
```

Comment:
Maybe a good time to rename to `processTransactions`? The difference has bothered me for a while
```suggestion
  readonly functionName: 'processTransactions';
```